config/webpack.prod.js

```diff
@@ -21,7 +21,7 @@ const webpackMerge = require('webpack-merge');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-const API_URL = process.env.API_URL || 'http://api.almighty.io/api/';
+const API_URL = process.env.API_URL || 'http://api.prod-preview.openshift.io/api/';
```

Comment:
This should be changed to 
`const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL;`

and the URL should not be set in Prod but you should set it in dev and use the same name.